panel/models/html.ts

```diff
@@ -123,7 +123,7 @@ export class HTMLView extends PanelMarkupView {
     return [...super.stylesheets(), html_css]
   }
 
-  protected rerender() {
```

Comment:
This one scares me and is probably not backward compatible. `rerender` used to be a panel only thing, seems like there's now an equivalent method in Bokeh that probably does something slightly different:

https://github.com/bokeh/bokeh/blob/fde15008802cf1fbb156b9f2af6ce434184bb934/bokehjs/src/lib/core/dom_view.ts#L58

Reply:
Yeah, was not sure about this one. If we still want to be backward compatible, can't we just rename the function?